mslib/utils/migration/config_before_nine.py

```diff
@@ -77,16 +77,16 @@ class MSUIDefaultConfig:
 
     # Predefined map regions to be listed in the corresponding topview combobox
     predefined_map_sections = {
-        "01 Europe (cyl)": {"CRS": "EPSG:4326",
+        "07 Europe (cyl)": {"CRS": "EPSG:4326",
```

Comment:
ToDo: The older MSS version needs an update and can't know the future. The change needs to become undone.

Reply:
https://github.com/Open-MSS/MSS/pull/2549